src/Salt/SiteBundle/Controller/CommentsController.php

```diff
@@ -133,7 +133,7 @@ public function upvoteAction(Comment $comment, UserInterface $user)
      *
      * @Method("DELETE")
      *
-     * @Security("is_granted('comment')")
+     * @Security("is_granted('comment', 'comment_view')")
```

Comment:
This should just have `comment` on the annotation.

`is_granted('comment_view')` should be on the `exportComentAction()` method.